### DISCUSSION TITLE
Image Source Metadata in Dockerfile

### DISCUSSION THREAD

**Discussion by coharms** [ghostfolio/ghostfolio#3476] [Ideas]
Not quite sure, if this is overkill, and I should have just opened a PR for this, but I would propose to include a label in the Dockerfile:

`LABEL org.opencontainers.image.source="https://github.com/ghostfolio/ghostfolio"`

It is a standard Docker Label referencing this repo within the metadata of the image.
For me, it would be useful, so [Renovate](https://docs.renovatebot.com/modules/datasource/docker/#description) can properly fetch the changelogs and display them before deployment. But as this is used by other Docker tools too, it could be beneficial to more users deploying/analyzing the self-hosted Docker version.
I could also open a corresponding PR, if wanted.

Thanks for consideration

**Comment by dtslvr** [comment]
Thank you for the suggestion @coharms. You are welcome to open a pull request for it.

**Reply by dtslvr** [reply]
Pull request: https://github.com/ghostfolio/ghostfolio/pull/3477
